addons/hr_attendance/__openerp__.py

```diff
@@ -18,20 +18,15 @@
     'data': [
         'security/ir_rule.xml',
         'security/ir.model.access.csv',
-        'hr_attendance_view.xml',
-        'hr_attendance_report.xml',
-        'wizard/hr_attendance_error_view.xml',
+        'views/hr_attendance_view.xml',
+        'views/hr_attendance_report.xml',
         'views/report_attendanceerrors.xml',
         'views/hr_attendance.xml',
-        'hr_dashboard.xml',
-    ],
-    'demo': ['hr_attendance_demo.xml'],
-    'test': [
-        'test/attendance_process.yml',
-        'test/hr_attendance_report.yml',
+        'views/hr_dashboard.xml',
+        'wizard/hr_attendance_error_view.xml',
     ],
+    'demo': ['demo/hr_attendance_demo.xml'],
```

Comment:
put demo file also in data directory [guidelines](https://github.com/odoo/odoo/blob/b9977ef5d30d98017b9f2d96bf151511b2eeafd2/doc/reference/guidelines.rst#directories)